windows/README.md

```diff
@@ -0,0 +1,34 @@
+# Windows Resources
+
+Windows agents can be wrapped with [winsw][] to run as Windows services.
+
+## Preparing the Jenkins agent
+
+To set up a Windows agent requires downloading the [winsw .NET4 executable](https://github.com/kohsuke/winsw/releases/).
+To keep paths as short as possible, we use the directory `C:\J` as our Jenkins home directory.
```

Comment:
Maybe add a little bit more explanation on why this is desirable, i.e. Windows doesn't like really long pathnames.